packages/snaps-rpc-methods/src/permitted/getCurrencyRate.ts

```diff
@@ -2,20 +2,17 @@ import type { JsonRpcEngineEndCallback } from '@metamask/json-rpc-engine';
 import type { PermittedHandlerExport } from '@metamask/permission-controller';
 import { rpcErrors } from '@metamask/rpc-errors';
 import type {
-  Currency,
+  AvailableCurrency,
   CurrencyRate,
   GetCurrencyRateParams,
   GetCurrencyRateResult,
   JsonRpcRequest,
 } from '@metamask/snaps-sdk';
-import { type InferMatching } from '@metamask/snaps-utils';
 import {
-  StructError,
-  create,
-  literal,
-  object,
-  union,
-} from '@metamask/superstruct';
+  currency as superstrucCurrency,
```

Comment:
Why does this need to be renamed? It also has a typo 😅

Reply:
Because `currency` is used in the RPC method implementation 😅 but I will rename the other one, seems more relevant